installer/engine_installer/engine_installer.go

```diff
@@ -14,8 +14,172 @@
 
 package main
 
-import "fmt"
+import (
+	"html/template"
+	"strings"
 
-func main() {
-	fmt.Println("hello world")
+	"context"
```

Comment:
Go imports are separated into 3 groups - standard, third-party and own pkgs, usually in that order and with a newline between each group.

In the above output:

* `log` is the only third-party pkg, hence the newline below it.
* The newline below `"github.com/gorilla/websocket"` is so you can separate the third-party group from the own packages.